addons/medical_treatment/functions/fnc_fullHealLocal.sqf

```diff
@@ -25,6 +25,9 @@ if ((["ace_fire"] call EFUNC(common,isModLoaded)) && {[_patient] call EFUNC(fire
     _patient setVariable [QEGVAR(fire,intensity), 0, true];
 };
 
+// Allow mods to heal
+[QGVAR(fullHealLocal), [_patient]] call CBA_fnc_localEvent;
```

Comment:
event calls this func - causes infinite recursion

Reply:
yeah why were they overwriting this when an even already existed....